Dockerfile

```diff
@@ -1,19 +1,21 @@
-FROM golang:1.14.4-alpine3.12 as build
+FROM golang:1.16.1-buster as build
```

Comment:
Could building in debian and running in alpine be problematic @jpfe-tid?

Reply:
I don't see any incompatibility yet, however build outputs this warning due to static:

```
# github.com/Telefonica/prometheus-kafka-adapter
/usr/bin/ld: /go/pkg/mod/github.com/confluentinc/confluent-kafka-go@v1.6.1/kafka/librdkafka_vendor/librdkafka_glibc_linux.a(rddl.o): in function `rd_dl_open':
(.text+0x1d): warning: Using 'dlopen' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/usr/bin/ld: /tmp/go-link-087041936/000026.o: in function `_cgo_26061493d47f_C2func_getaddrinfo':
/tmp/go-build/cgo-gcc-prolog:58: warning: Using 'getaddrinfo' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
```

I'll try in a following PR using net's go implementation to remove it.